CHANGELOG.md

```diff
@@ -0,0 +1,92 @@
+# Changelog
+All notable changes to `frunk` and its subcrates will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]:
+### Added
+- This CHANGELOG file. :wink:
+
+## [0.2.0] - 2018-04-20
+### Added
+- `frunk` now adheres to semantic versioning!
+- Inherent method wrappers for many trait methods on HList and Coproduct.
+- Re-exports for a variety of essential items at the root level of `frunk`.
+- Module `frunk::prelude` for extension traits and similar.
+- Methods `Coproduct::embed` and `Coproduct::subset`.
+- `...tail` argument in each of the HList and Coproduct macros.
+- The `Func` trait, an alternative to `Fn` that can be implemented on stable.
+- The `Poly` wrapper type and `poly_fn!` macro, which use `Func` to provide
+  order-free mapping.
+- `map_repr` and `map_intern` for `Generic`, convenience methods for
+  temporarily converting an object into its generic representation or
+  a similar type.
+
+### Changed
```

Comment:
I feel like the improvements and corrections to documentation from you and @Centril deserves a call-out.